pkg/util/tracing/tracer.go

```diff
@@ -1165,6 +1165,7 @@ child operation: %s, tracer created at:
 	if traceID == 0 {
 		traceID = tracingpb.TraceID(randutil.FastInt63())
 	}
+	logtags.AddTag(ctx, "trace", fmt.Sprintf("%d", traceID))
```

Comment:
note to self for later: we need to return this ctx.

Reply:
nice catch, done